psi4/src/export_plugins.cc

```diff
@@ -61,14 +61,16 @@ int py_psi_plugin_load(std::string fullpathname)
     filesystem::path pluginPath(fullpathname);
     std::string uc = to_upper_copy(pluginPath.stem());
 
-    // Make sure the plugin isn't already loaded.
-    if (plugins.count(uc) == 0) {
-        plugins[uc] = plugin_load(fullpathname);
-        outfile->Printf("%s loaded.\n", fullpathname.c_str());
-        ret = 1;
-    } else
-        ret = 2;
-
+//    // Make sure the plugin isn't already loaded.
+//    if (plugins.count(uc) == 0) {
+//        plugins[uc] = plugin_load(fullpathname);
+//        outfile->Printf("%s loaded.\n", fullpathname.c_str());
+//        ret = 1;
+//    } else
+//        ret = 2;
+    plugins[uc] = plugin_load(fullpathname);
```

Comment:
We're doing some work to make plugins more flexible. To allow plugins to work in PsiAPI mode and still go through the plugin machinery, they needed some options commands _every_ time initialized that are in `libplugin/plugin_load.cc`. As soon as you go a step further with plugins and drop the plugin machinery for a plugin itself exported to python via pybind11, these changes aren't necessary, and we actually plan to revert them.